src/main/scala/mesosphere/marathon/core/task/tracker/impl/InstanceTrackerDelegate.scala

```diff
@@ -97,14 +97,16 @@ private[tracker] class InstanceTrackerDelegate(
         val effectF = (instanceTrackerRef ? update)
           .mapTo[InstanceUpdateEffect]
           .transform {
-            case s@Success(_) => logger.info(s"Completed processing instance update ${update.operation.shortString}"); s
-            case f@Failure(e: AskTimeoutException) => logger.error(s"Timed out waiting for response for update $update", e); f
-            case f@Failure(t: Throwable) => logger.error(s"An unexpected error occurred during update processing of: $update", t); f
+            case s @ Success(_) =>
+              logger.info(s"Completed processing instance update ${update.operation.shortString}"); s
+            case f @ Failure(e: AskTimeoutException) =>
+              logger.error(s"Timed out waiting for response for update $update", e); f
+            case f @ Failure(t: Throwable) => logger.error(s"An unexpected error occurred during update processing of: $update", t); f
```

Comment:
Do you think `NonFatal(t)` is better here?

Reply:
That's just a reformatting but yes 🙂

Reply:
Why? Would you treat `NonFatal(t)` exceptions differently here? This whole `transform` part exists for logging purposes only.

Reply:
Agree with @zen-dog ; we're not intercepting exceptions, we're logging them.

Matching all throwables is fine. Let's do `f @ Failure(t)` though?